src/main/java/com/epam/reportportal/auth/integration/ldap/LdapUserReplicator.java

```diff
@@ -89,7 +90,7 @@ public User replicateUser(String name, DirContextOperations ctx, Synchronization
 			newUser.setRole(UserRole.USER);
 			newUser.setExpired(false);
 
-			newUser.setDefaultProject(generatePersonalProject(newUser));
+			generatePersonalProject(newUser).getUsers().stream().findFirst().ifPresent(it -> newUser.getProjects().add(it));
```

Comment:
probably it's better to split logic of project adding to the user
cause from the first watch it doesn't really clear to understand what is going on here.